doc/sphinxman/source/plugin_snsmp2.rst

```diff
@@ -0,0 +1,99 @@
+.. #
+.. # @BEGIN LICENSE
+.. #
+.. # Psi4: an open-source quantum chemistry software package
+.. #
+.. # Copyright (c) 2007-2017 The Psi4 Developers.
+.. #
+.. # The copyrights for code used from other parties are included in
+.. # the corresponding files.
+.. #
+.. # This file is part of Psi4.
+.. #
+.. # Psi4 is free software; you can redistribute it and/or modify
+.. # it under the terms of the GNU Lesser General Public License as published by
+.. # the Free Software Foundation, version 3.
+.. #
+.. # Psi4 is distributed in the hope that it will be useful,
+.. # but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.. # GNU Lesser General Public License for more details.
+.. #
+.. # You should have received a copy of the GNU Lesser General Public License along
+.. # with Psi4; if not, write to the Free Software Foundation, Inc.,
+.. # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+.. #
+.. # @END LICENSE
+.. #
+
+.. include:: autodoc_abbr_options_c.rst
+
+.. index:: SNS-MP2
+
+Spin-Network-Scaled MP2 (SNS-MP2) by D. E. Shaw
+===============================================
+
+.. codeauthor:: D. E. Shaw Research
+.. sectionauthor:: Shannon E. Houck
+
+.. image:: https://img.shields.io/badge/home-sns--mp2-5077AB.svg
+   :target: https://github.com/DEShawResearch/sns-mp2
+
+.. raw:: html
+
+   <br>
+
+.. image:: https://img.shields.io/badge/docs-latest-5077AB.svg
+   :target: https://github.com/DEShawResearch/sns-mp2/blob/master/README.md
+
+This plugin is an implementation of the SNS-MP2 algorithm developed by McGibbon 
+et. al. [McGibbon:2017:161725]_. The SNS-MP2 method uses neural networking to 
+improve the accuracy of MP2 (:ref:`dfmp2`) interaction energies for dimer molecules.
```

Comment:
It's probably worth mentioning that their code is distributed under the [BSD 2-clause](https://opensource.org/licenses/BSD-2-Clause) license, because it's different from our LGPL license.  It's not a big deal, as BSD-2 is more permissive than LGPL, but it could help out some legal folks.

Reply:
I've added this info to the docs. Thank you!